wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConfigImpl.java

```diff
@@ -108,6 +121,12 @@ public Config add(String name, String value) throws GitException {
     public Config unset(String name) throws GitException {
         ConfigKey key = parseName(name);
         repository.getConfig().unset(key.section, key.subsection, key.name);
+        try {
+            this.repository.getConfig().save();
+        } catch (IOException e) {
+            LOG.debug(String.format(ERROR_FAIL_SAVE_CONFIG, e.getCause().getMessage()), e.getCause());
```

Comment:
note: as now you're sending the exception in the GitException (before it was dropped) the logger call is optional